src/main/java/sg/ncl/exceptions/WebServiceRuntimeException.java

```diff
@@ -0,0 +1,12 @@
+package sg.ncl.exceptions;
+
+
+/**
+ * @author Te Te
+ */
+public class WebServiceRuntimeException extends Exception {
+
+    public WebServiceRuntimeException(String message) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUndocumentedApi)